Introductory Tests/UDP_basic.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.            # Defining additional parameters
</pre><p>the parameters also don't need to be redefined everytime, should go into constructor</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasmusmarak/67P-Swarm/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>